content/admin/org_settings/OrgNotifications.md

```diff
@@ -73,3 +73,6 @@ Administrators automatically receive the following notifications for high-level
 
 <a href="assets/images/AdminNotificationEmail.png" rel="lightbox" title="Email notification for administrators"><img class="thumbnail" src="assets/images/AdminNotificationEmail.png"/></a>
 
+## Feature Notifications 
```

Comment:
Suggestions:
"Some features inherently require notification, in which case once an admin has enabled the feature, Contrast will automatically notify the relevant parties (whether a user or an administrator). These notifications are not controlled in the Notifications page. Contrast requires notifications for features including vulnerability status approval and policy management."  [That's all I can think of right now - we can add another example later if need be.]